commands/serverless.go

```diff
@@ -114,9 +114,28 @@ the entire packages are removed.`, Writer)
 
 // RunServerlessInstall performs the network installation of the 'nim' adjunct to support serverless development
 func RunServerlessInstall(c *CmdConfig) error {
-	credsLeafDir := hashAccessToken(c)
-	serverless := c.Serverless()
-	status := serverless.CheckServerlessStatus(credsLeafDir)
+	var (
+		serverless   do.ServerlessService
+		credsLeafDir string
+		status       error
+	)
+
+	// When building the snap package, we need to install the serverless plugin
+	// without a fully configured and authenticated doctl. So we only fully init
+	// the service if SNAP_SANDBOX_INSTALL is not set.
+	_, isSnapInstall := os.LookupEnv("SNAP_SANDBOX_INSTALL")
+	if isSnapInstall {
+		serverlessDir := os.Getenv("OVERRIDE_SANDBOX_DIR")
+		serverless = do.NewServerlessService(nil, serverlessDir, "")
```

Comment:
I believe the NewServerlessService function will interrogate OVERRIDE_SANDBOX_DIR itself and ignore the argument, so perhaps it is unnecessary to do it twice.   Just a nit.  it should be harmless.